src/main/entities/minigames/Maze.java

```diff
@@ -1,11 +1,11 @@
 package entities.minigames;
 
 import entities.Interactable;
-import entities.minigames.Timer;
 import entities.Unaffordable;
 import entities.interfaces.Moveable;
 import entities.characters.*;
 import entities.weapons.*;
+import usecases.Timing;
```

Comment:
I was talking with shehzaad and he mentioned Java has a built-in multithread timer. Any reason you chose to make your own?

Reply:
It illustrates the CA stuff